README.md

```
@@ -34,11 +34,11 @@ $ go install github.com/ibnaleem/gosearch@latest
 ```
 ### Unix:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)